manifests/init.pp

```diff
@@ -176,6 +176,9 @@
 #
 # $manage_squid::               boolean controls whether squid configuration is managed or manual
 #
+# $lazy_enable::                This controls whether redirect is enabled
```

Comment:
The explanation is from server.conf:

# enabled:
#   This controls whether redirect is enabled